.vscode/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
should this file/folder be here in the repo? Do we already have a meaningful .gitignore?

Reply:
Debatable, I thought yes; it's the build command definition for VS Code. I can remove/ignore it if desired.
Yes, we have a default gitignore, it seems meaningful so far.

Reply:
OK, I removed them